examples/sdkservice.go

```diff
@@ -47,21 +47,21 @@ func main() {
 		AgentType:      nimbleos.NsAgentTypeNone,
 	}
 
-	vol,_ := groupService.GetVolumeService().CreateVolume(newVolume)
+	vol, _ := groupService.GetVolumeService().CreateVolume(newVolume)
 	fmt.Printf("Fake volume %+v \n", vol)
 
 	// Get real service
-	groupService, _  = getRealService("10.21.4.79", "admin", "admin", "v1", true)
+	groupService, _ = getRealService("1.1.1.1", "xxx", "xxx", "v1", true, false)
```

Comment:
This suggestion is not meant for this PR, but perhaps it can be taken up as a subsequent change if others think it worthwhile....

It seems inconvenient to have to edit the source to be able to run the examples.  Perhaps the host, user name, and password could be read from the environment instead.

The same issue exists in the test cases, for example volume_service_test.go.  Reading from the environment seems sensible for those, too.  Also the test cases should be changed to use the non-deprecated function.